### PR TITLE
added popup menu

### PR DESCRIPTION
**Pull request by AtillaTahak**
-added js file
-added HTML code for modal
-added icon modal menu for the button
-added CSS code for modal menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,68 +114,109 @@ <h2>I’am a Software Developer</h2>
...
                         <li>Boostrap</li>
                         <li>Ruby</li>
                     </ul>
+                    <button data-model="3" class="seeProject modal-button">See Project</button>
+
                 </div>
             </div>
 
+        </section>
+        <section class="recent-work-model">
```
Comment (Mihndim2020) [inline index.html:174]: - [ ] Please make sure your popup stays within the view area, as this a requirement for this project. As a front end developer, you need to stick to the design as much as possible. Please kindly fix that. See images below.

***The design***
atillaD

***Your implementation***

atillaI

- [ ] Please make sure the components in the popup and laid out as in the design(see image above)

***Your implementation***
attillaI2

File: index.html [hunk 2]
```diff
@@ -113,68 +114,109 @@ <h2>I’am a Software Developer</h2>
...
+                <div class="recent-work-p">
+                    <p id="modal-p">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+                        Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer
+                        took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of
+                        the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text
+                        ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with
+                        the releax map lapora verita.</p>
+
+                    <button>See Live<span class="icon1"></span></button>
+                    <button>See Source<span class="icon2"></span></button>
```
Comment (Mihndim2020) [inline index.html]: - [ ] This should be added to the JavaScript object as it is a requirement for the this project. Please do revisit your project requirements. See image below.

req

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Linters are passing
:heavy_check_mark: The project is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION



# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (AtillaTahak) [reply index.html]: image
this is a javascript object pls check my code

File: index.html (hunk 1 above)
Comment (AtillaTahak) [reply index.html:174]: I don't get it what do you mean view area

**Review by dasileker — APPROVED** [review]
### **Hi @AtillaTahak,** 

### **Congratulations! 🎉Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 


Approved
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**